test/promql/tests/test_controller_reconciliation_errors.yaml

```diff
@@ -0,0 +1,91 @@
+evaluation_interval: 1m
+
+rule_files:
+  - ../extracted-rules.yaml
+
+tests:
+
+  # Alerted test cases
+  - interval: 1m
+    input_series:
+      # Controller with continuous errors greater than 1% so this is alerted
```

Comment:
Can you include some more variations of time series values such as large spikes of errors mixed with pauses in reconcile activity?